### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by melissamurphy**
navbar

### CODE REVIEW

File: client/components/course-search-result.jsx [hunk 1]
```diff
@@ -27,7 +28,11 @@ export default class CourseSearchResult extends React.Component {
             </div>
           </div>
           <button
-            onClick={() => this.setViewAndCourseId('myLessons', courseId)}
+            onClick={() => {
+              this.setViewAndCourseId('myLessons', courseId);
+              this.props.setBackPage(backPage);
+            }
+            }
```
Comment (jasmine-gallardo) [inline client/components/course-search-result.jsx]: Have the onClick only call one function and call setBackPage() in the setViewAndCourseId() method. Then just rename that method with a clear/succinct name.

File: client/components/course.jsx [hunk 2]
```diff
@@ -14,10 +14,15 @@ export default class Course extends React.Component {
...
+    const backPage = this.props.currentPage;
     return (
       <div className="w-100 d-flex justify-content-center">
         <button
-          onClick={() => this.setViewAndCourseId('myLessons', courseId)}
+          onClick={() => {
+            this.setViewAndCourseId('myLessons', courseId);
+            this.props.setBackPage(backPage);
+          }
+          }
```
Comment (jasmine-gallardo) [inline client/components/course.jsx]: Have the onClick only call one function and call setBackPage() in the setViewAndCourseId() method. Then just rename that method with a clear/succinct name.

**Review by jasmine-gallardo — CHANGES_REQUESTED** [review]

File: client/components/course-search-result.jsx (hunk 1 above)
Comment (melissamurphy) [reply client/components/course-search-result.jsx]: Took everything in the callback code block and put it into new method 'setNextPage'

File: client/components/course.jsx (hunk 2 above)
Comment (melissamurphy) [reply client/components/course.jsx]: Took everything in the callback code block and put it into new method 'setNextPage'

**Review by jasmine-gallardo — APPROVED** [review]

**Pull request merged**